content/200-orm/100-prisma-schema/10-overview/04-location.mdx

```diff
@@ -0,0 +1,105 @@
+---
+title: Schema location
+metaTitle: Prisma Schema Location and Configuration
+metaDescription: Documentation regarding proper location of Prisma Schema including default naming and multi-file schema.
+---
+
+The default name for the Prisma schema is a single file `schema.prisma` in your `prisma` folder. When your schema is named like this, the Prisma CLI will detect it automatically.
+
+> If you are using the [`prismaSchemaFolder` preview feature](#multi-file-prisma-schema) any files in the `prisma/schema` directory are detected automatically.
+
+## Prisma Schema location
+
+The Prisma CLI looks for the Prisma schema in the following locations, in the following order:
+
+1.  The location specified by the [`--schema` flag](/orm/reference/prisma-cli-reference), which is available when you `introspect`, `generate`, `migrate`, and `studio`:
+
+    ```terminal
+    prisma generate --schema=./alternative/schema.prisma
+    ```
+
+2.  The location specified in the `package.json` file (version 2.7.0 and later):
+
+    ```json
+    "prisma": {
+      "schema": "db/schema.prisma"
+    }
+    ```
+
+3.  Default locations:
+
+    - `./prisma/schema.prisma`
+    - `./schema.prisma`
+
+The Prisma CLI outputs the path of the schema that will be used. The following example shows the terminal output for `prisma db pull`:
+
+```no-lines
+Environment variables loaded from .env
+//highlight-next-line
+Prisma Schema loaded from prisma/schema.prisma
+
+Introspecting based on datasource defined in prisma/schema.prisma …
+
+✔ Introspected 4 models and wrote them into prisma/schema.prisma in 239ms
+
+Run prisma generate to generate Prisma Client.
+```
+
+## Multi-file Prisma Schema
+
+:::tip
+
+Multi-file Prisma Schema is available via the `prismaSchemaFolder` preview feature in Prisma versions 5.15.0 and later.
+
+:::
+
+To use multiple Prisma schema files, add a `schema` folder inside of your current `prisma` directory. With the `prismaSchemaFolder` Preview feature enabled, you can add as many files as you want to the `prisma/schema` directory.
```

Comment:
This could benefit from a code block showing the project structure with a `prisma/schema` folder with some `.prisma` files.

Reply:
````suggestion
To use multiple Prisma schema files, add a `schema` folder inside of your current `prisma` directory. With the `prismaSchemaFolder` Preview feature enabled, you can add as many files as you want to the `prisma/schema` directory.
```
my-app/
├─ ...
├─ prisma/
│  ├─ schema/
│  │  ├─ post.prisma
│  │  ├─ schema.prisma
│  │  ├─ user.prisma
├─ ...
```
````